src/styles/blocks/catalog.scss

```diff
@@ -0,0 +1,139 @@
+@import "../utils/variables";
+@import "../utils/mixins.scss";
+@import "../utils/reset.scss";
```

Comment:
all your imoprt's is better to put inside main.scss, this will solve the duplication problem
connect all your utilities before connecting the blocks,